### PR TITLE
Discover & Movie Pages

### PR DESCRIPTION
**Pull request by gabichuelas**
- Discover page has button to 'Find Top Rated Movies' and input search field for movie keywords.
- 'Find Top Rated Movies' links to GET MoviesController#index, where there is a Faraday connection to the Movie DB API to get 2 pages worth of top rated movies. _This will be refactored in a future PR, probably using a PORO for the Movie DB API work_
- Movies#index view shows a list of the 40 top rated movies along with their vote_average score. 
- Each movie title is a link (which is not routed yet) - will go to Movies Detail Page ([issue #10](https://github.com/rrabinovitch/viewing_party/issues/10))

- All tests pass.

### CODE REVIEW

File: app/controllers/movies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      req.params['api_key'] = ENV["MOVIE_DB_API_KEY_V3"]
+      req.params['language'] = 'en-US'
+      req.params['page'] = '2'
+    end
+    
+    json1 = JSON.parse(page1.body, symbolize_names: true)
+    json2 = JSON.parse(page2.body, symbolize_names: true)
+
+    @movies_pg1 = json1[:results]
+    @movies_pg2 = json2[:results]
```
Comment (rrabinovitch) [inline app/controllers/movies_controller.rb:22]: it doesn't look like these instance variables are referenced outside of this file, so the `@` can probably be removed, no?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,6 +9,11 @@
   </head>
 
   <body>
+    <header>
+      <% if current_user %>
+        <h3>Welcome, <%= current_user.username %></h3>
+      <% end %>
+    </header>
```
Comment (rrabinovitch) [inline app/views/layouts/application.html.erb:16]: good call w moving this into `application` and adding the conditional

File: app/views/movies/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<%= render '/partials/movie_controls' %>
```
Comment (rrabinovitch) [inline app/views/movies/index.html.erb:1]: yay partials!

**Review by rrabinovitch — APPROVED** [review]
left a comment to consider, but still looks good to merge. woot woot!

File: app/controllers/movies_controller.rb (hunk 1 above)
Comment (gabichuelas) [reply app/controllers/movies_controller.rb:22]: whoops, yeah, they were originally instance variables bc I planned on using them both before I realized I could combine them. Will fix for next push.

**Pull request merged**